tests/pdctl/store/store_test.go

```diff
@@ -276,6 +277,17 @@ func (s *storeTestSuite) TestStore(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(strings.Contains(string(output), "Unknown state: Invalid_state"), IsTrue)
 
+	// store undelete <store_id> command
+	args = []string{"-u", pdAddr, "store", "undelete", "1"}
```

Comment:
revote-delete? revote-offline?

Reply:
Do you mean revoke or revert?

Reply:
I think `revoke` is better than `undelete`